git-tar/function/ops.go

```diff
@@ -493,3 +493,75 @@ func getShortSHA(sha string) string {
 	}
 	return sha[:7]
 }
+
+func reportStatus(status *sdk.Status, SCM string) {
+	if SCM == "github" {
+		reportGitHubStatus(status)
+	} else {
+		reportGitLabStatus(status)
+		for i := range status.CommitStatuses {
+			delete(status.CommitStatuses, i)
+		}
+	}
+}
```

Comment:
the same suggestion as in buildshiprun comment

Reply:
Also done